setup.py

```diff
@@ -48,5 +48,5 @@
     keywords="adafruit ble bluefruit bluetooth micropython circuitpython",
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
-    py_modules=["adafruit_bluefruit_connect"],
+    py_packages=["adafruit_bluefruit_connect"],
```

Comment:
I think maybe it needs to be `packages=["adafruit_bluefruit_connect"],` note the lack of leading `py_`.

That seems to be the main difference I can spot as compared to here: https://github.com/adafruit/Adafruit_Blinka_Displayio/blob/master/setup.py

I went ahead and made this change in my local copy and tested again. After changing this and doing `pip3 install .` I can now successfully complete those imports from python3 shell in other arbitrary locations.

Reply:
You're right, thanks for catching this! I was testing this with `pip3 install -e ...`, which worked, but not because I had fixed things.